Flabbergasted-Flamingos/Bangazon-admin-app
# addded test to check to make sure correct template renders

# Description
This pull request is the final test for #1 

## Number of Fixes
1 fix, final test needed for ticket to be closed

## Related Ticket(s)
#1 

## Problem to Solve
Now you can run tests to make sure that the correct template loads for the employee list

## Proposed Changes
Added test def at end of employees_list_tests.py

## Expected Behavior
No behaviour other than now 5 tests should pass, or all 3 of mine

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_final_test
python manage.py runserver
```
1. make sure employee list is still there on server
2. Run tests
```
python manage.py test
```
make sure all tests pass!

## Testing

[x ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x ] I certify that all existing tests pass
